build is failing with latest nbconvert update

**Describe the bug**
the textbook build fails after setting up a new/clean environment. more specifically, when `jupyter-book build` is run it fails with error `jinja2.exceptions.TemplateNotFound: base.html.j2`:

build log: https://travis-ci.com/github/qiskit-community/qiskit-textbook/builds/183710951

build error:
```
jupyter-book build ./ --overwrite
  0%|          | 0/58 [00:00<?, ?it/s]
Convert and copy notebook/md files...
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.7.1/bin/jupyter-book", line 8, in <module>
    sys.exit(main())
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/jupyter_book/main.py", line 30, in main
    commands[args.command]()
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/jupyter_book/commands/build.py", line 50, in build
    local_build=local_build, execute=execute, overwrite=overwrite, clear_output=clear_output)
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/jupyter_book/build.py", line 287, in build_book
    author=html_author
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/jupyter_book/page/page.py", line 246, in page_html
    html, resources = exp.from_notebook_node(ntbk, resources=output_resources)
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/nbconvert/exporters/html.py", line 122, in from_notebook_node
    return super().from_notebook_node(nb, resources, **kw)
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/nbconvert/exporters/templateexporter.py", line 382, in from_notebook_node
    output = self.template.render(nb=nb_copy, resources=resources)
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/home/travis/virtualenv/python3.7.1/lib/python3.7/site-packages/jupyter_book/page/templates/html.tpl", line 2, in top-level template code
    {% from 'celltags.tpl' import celltags %}
  File "/home/travis/virtualenv/python3.7.1/share/jupyter/nbconvert/templates/compatibility/basic.tpl", line 2, in top-level template code
    {%- extends 'classic/index.html.j2' -%}
  File "/home/travis/virtualenv/python3.7.1/share/jupyter/nbconvert/templates/classic/index.html.j2", line 1, in top-level template code
    {%- extends 'base.html.j2' -%}
jinja2.exceptions.TemplateNotFound: base.html.j2
Makefile:34: recipe for target 'build' failed
make: *** [build] Error 1
```

**To Reproduce**
Steps to reproduce the behavior:
1. set up a new/clean environment
1. run `make build` or `make book`
1. see error

**Expected behavior**
for the textbook to build cleanly and successfully

**Addition info**
this failure coincides with the recent release of `nbconvert` 6.0 which is getting installed when downloading dependencies and setting up a new environment. downgrading to the previous `nbconvert` release (i.e., 5.6.x) resolves the error

--- a/requirements-dev.txt
+++ b/requirements-dev.txt
@@ -1,3 +1,4 @@
+nbconvert==5.6.1
 jupyter-book==0.6.5
 beautifulsoup4
 

